src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -455,10 +455,9 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
       await this.props.indexPatterns.clearCache();
     }
 
-    const objects = await savedObjectsClient.bulkGet(selectedSavedObjects);
-    const deletes = objects.savedObjects.map((object) =>
-      savedObjectsClient.delete(object.type, object.id, { force: true })
-    );
+    const deletes = selectedSavedObjects
+      .filter((object) => !object.meta.hiddenType)
+      .map((object) => savedObjectsClient.delete(object.type, object.id, { force: true }));
```

Comment:
This was performing a `bulkGet` for no reason, as we already have all the info we need on the client side to perform the deletes.

Reply:
Another use case for `bulkDelete` (context https://github.com/elastic/kibana/issues/30503)